lua/core/formatter.lua

```diff
@@ -53,7 +53,7 @@ if not status_ok then
   return
 end
 
-formatter.setup {}
+formatter.setup { formatter.config }
```

Comment:
@ChristianChiarulli take a look at this PR and let me know if you like it or not
also without this line, the formatters wouldn't have been set